app/validators/password_validator.rb

```diff
@@ -0,0 +1,15 @@
+class PasswordValidator < ActiveModel::Validator
+  PASSWORD_FORMAT = /\A
+    (?=.*\d)    # a number
+    (?=.*[a-z]) # a lower case letter
+    (?=.*[A-Z]) # an upper case letter
+  /x
+  # (?=.{8,})   # 8 or more characters
```

Comment:
Just checking, is this intentionally commented out?

Reply:
Yes, for the time being we do not enforce symbols, just leaving it there for reference ahead of pen testing